### ISSUE TITLE
Support writable computeds

### ISSUE THREAD

**Issue by rubenduiveman** [facing-dev/vue-facing-decorator#23]
At this moment, `get` functions are converted to a computed. However, I'd very much like to be able to use [writable computeds](https://vuejs.org/guide/essentials/computed.html#writable-computed.) by combining `get` with ` set`. 

So what I propose is that this
```typescript
@Prop() private value;

private get model() {
  return this.value;
}

private set model(to: string) {
  this.$emit('update:modelValue", to);
}
```

gets converted to this
```typescript
computed: {
  model: {
    get() {
      return this.value;
    },
    set() {
      this.$emit('update:modelValue", to);
    }
  }
}
```

**Comment by ruojianll** [comment]
We have a `@Model` decorator to do that.

**Comment by ruojianll** [comment]
https://facing-dev.github.io/vue-facing-decorator/#/en/class-component/model/model

**Comment by rubenduiveman** [comment]
Yes, that's if I want to update a v-model. But how should this work if I want to update a VueX store property instead? I want to use the `get` to read the store value, and the `set` to update the store. I don't think this is possible with `@Model`

**Comment by ruojianll** [comment]
Supported in v2.1.8.
And vanilla getter for no cached computed.

https://facing-dev.github.io/vue-facing-decorator/#/en/class-component/accessor/accessor?id=writable

**Comment by rubenduiveman** [comment]
Wow super fast! Thanks! I'll give it a spin!
